pkg/schemas/stream_gen.go

```diff
@@ -3,11 +3,14 @@
 package schemas
 
 import (
+	"log"
+
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 // ShowStreamSchema represents output of SHOW query for the single Stream.
+// TODO: merge with desc schema
```

Comment:
next pr ?

Reply:
Nope, I think we'll leave this in case of some changes on the Snowflake side. I don't think we have coupled structs anywhere.

Reply:
yeah, let;s not merge these structs